resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -54,17 +54,17 @@ package object config {
       .stringConf
       .createWithDefault(s"spark-executor:$sparkVersion")
 
-  private[spark] val KUBERNETES_CA_CERT_FILE =
-    ConfigBuilder("spark.kubernetes.submit.caCertFile")
+  private[spark] val KUBERNETES_APISERVER_CA_CERT_FILE =
+    ConfigBuilder("spark.kubernetes.apiserver.caCertFile")
       .doc("""
           | CA cert file for connecting to Kubernetes over SSL. This
           | file should be located on the submitting machine's disk.
         """.stripMargin)
       .stringConf
       .createOptional
 
-  private[spark] val KUBERNETES_CLIENT_KEY_FILE =
-    ConfigBuilder("spark.kubernetes.submit.clientKeyFile")
+  private[spark] val KUBERNETES_CLIENT_APISERVER_KEY_FILE =
+    ConfigBuilder("spark.kubernetes.apiserver.clientKeyFile")
```

Comment:
Actually let's just do `submission` for the SSL one since that makes it consistent.